### PR TITLE
🐛 Add default url in case no url is found

### PR DESCRIPTION
**Pull request by GertjanReynaert**
In the current implementation if no valid `https` link is found for a package, the `url` key won't be present in the json output for this package.

Instead of omitting the `url` key, this PR will use a default npm url instead. This way, when you need info on a package when you're looking at the licenses json file, it's easier to get a lead to more information of a package.

### CODE REVIEW

**Comment by gboston** [comment]
tests would be nice :)

**Comment by gboston** [comment]
Example of some lightweight testing with tap: https://github.com/inthepocket/fastify-typeorm-plugin/blob/master/test.js

**Comment by GertjanReynaert** [comment]
@gboston agreed, if there was already a test suite.

Since there's none, I'd not block this pr and implement that in a different PR

**Comment by eliaslecomte** [comment]
Will you fix the return type of that function?

`export function getPackageInfo(pkg: PackageJSON): {
  version: PackageJSON['version'];
  license: PackageJSON['license'];
  url: undefined | string;
} `

To be honest, I don't think we need to define the return type here. What does code that uses this when url is undefined? Does that needs changing?

**Comment by GertjanReynaert** [comment]
When `url` was undefined, the `url` key would just be omitted from the json output. Now instead of being omitted, it'll be present with the default npm url.

**Review by eliaslecomte — APPROVED** [review]

**Comment by eliaslecomte** [comment]
Who will release this to npm after merging? @thibmaek?

**Pull request merged**
